synfig-studio/src/gui/duckmatic.h

```diff
@@ -114,6 +114,12 @@ class BezierDrag_Default : public BezierDrag_Base
 	void bezier_drag(Duckmatic* duckmatic, const synfig::Vector& vector);
 };
 
+struct Guide
+{
+	synfig::Point point;
+	synfig::Angle::rad angle;
+};
+
```

Comment:
My only doubt in this PR here is if Guide should be defined in duckmatic.h lol

Reply:
Hmm I believe this was where the previous guide lists were also defined.